features/src/test/java/com/openapi/forge/RunCucumberTest.java

```diff
@@ -13,4 +14,9 @@
 @SelectClasspathResource("com/openapi/forge")
 @ConfigurationParameter(key = PLUGIN_PROPERTY_NAME, value = "pretty")
 @ConfigurationParameter(key = GLUE_PROPERTY_NAME, value = "com.openapi.forge")
+// dont run one particular scenario with a get request with body.
```

Comment:
Can we add a bit of explanation here? At least linking to the discussion here: https://github.com/ScottLogic/openapi-forge/issues/182

It would be good to be clear that we are choosing not to support it because it is non-standard behaviour, not just as a hack because we can't get it working.